crates/bevy_dev_tools/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+#![warn(missing_docs)]
+//! This crate provides additional utilities for the [Bevy game engine](https://bevyengine.org),
+//! aimed at improving developer experience.
```

Comment:
```suggestion
//! focused on interactive in-app debugging. 
```

Reply:
Do we want this crate to be only for debugging tools? There *might* be some other tools, that would improve workflow but are not designed for debugging.